superset/connectors/sqla/views.py

```diff
@@ -361,6 +371,43 @@ def create(self):
         db.session.commit()
         return jsonify(success=True)
 
+    @expose('/<pk>', methods=['GET', 'POST'])
+    @has_access
+    def edit(self, pk):
+        alert = self.datamodel.get(pk)
+        if request.method == 'POST':
+            data = request.get_json()
+            modified_alert = self.datamodel.get(data['edit_id'])
+            modified_alert.name = data['name']
```

Comment:
@qv-pat Is there a standard way of updating these fields that is used in the Superset code base? This code looks very repetitive. You could try using getattr and loop through the fields: http://effbot.org/zone/python-getattr.htm.

Reply:
Superset leverages the default flask editor for all of their other models. I agree that there are probably more succinct methods of doing this. Doing a get/setattr is probably a good way to go about it. Good idea.